src/include/com/amazonaws/kinesis/video/webrtcclient/NullableDefs.h

```diff
@@ -0,0 +1,94 @@
+/*
+ * Header file to define nullable values for common data types
+ */
+
+#ifndef __KINESIS_VIDEO_WEBRTC_CLIENT_NULLABLEDEFS__
+#define __KINESIS_VIDEO_WEBRTC_CLIENT_NULLABLEDEFS__
+
+#pragma once
+
+#ifdef  __cplusplus
+extern "C" {
+#endif
+
+////////////////////////////////////////////////////
+// Public headers
+////////////////////////////////////////////////////
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wgnu-zero-variadic-macro-arguments"
+#pragma clang diagnostic pop
+
+
+
+#define SET_DEFAULT_AS_NULL(a) \
+    do { \
+        a.isNullable = TRUE; \
+        a.value = 0; \
+    } while(FALSE)
+
+#define IS_VALUE_NULL(a) ((a.isNullable) & TRUE)
+
```

Comment:
can you add a macro for getting the value?

Reply:
+1

Reply:
Do we need that? What would be the purpose? The value can just be accessed right?

Reply:
How about

NULLABLE_CHECK_VALUE(a) // returns TRUE if value is not null
NULLABLE_SET_VALUE(a, x) // Sets the value and sets not null
NULLABLE_SET_EMPTY(a) // Sets the nullable to empty?

Reply:
Oh and NULLABLE_GET_VALUE ?

Reply:
We can do that. Let me add the set part.

Reply:
Done

Reply:
Done